### PR TITLE
Tuto for psy

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: frontend/src/components/Tutorial/Steps/students.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    content: "Les étudiants enregistrés apparaîtront ici, vous pourrez les supprimer en cas d'erreur ou modifier leurs informations.",
+  },
+  {
+    placement: 'auto',
+    target: '#students-table',
+    shouldSkip: () => agent.Patient.get().then(students => students.length === 0),
+    content: 'Ce tableau liste tous les étudiants que vous avez enregistré, le statut de leur dossier et le nombre total de séances que vous avez déclaré pour lui/elle.',
+  },
+  {
+    placement: 'auto',
```
Comment (sandrica89) [inline frontend/src/components/Tutorial/Steps/students.js:28]: I tried to use 'top-end' and 'right-start' placement here but it doesn't seem to work :(

Comment (desoindx) [reply frontend/src/components/Tutorial/Steps/students.js:28]: it's working for me, we can look together

**Comment by sandrica89** [comment]
I have an issue in the review app : on my first load, I have an issue when I go to the login page. The console shows this error : `Uncaught TypeError: Cannot read properties of undefined (reading 'hasSeenTutorial')`.

**Comment by sandrica89** [comment]
If #431 is merged before, condition on profil step should be added on "see public profile" button.

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ function App() {
           <Switch>
             <Route exact path="/psychologue/logout" component={Logout} />
             <Route exact path="/psychologue/login/:token?" component={Login} />
-            {user && <Route path="/psychologue/" component={PsychologistRouter} />}
```
Comment (sandrica89) [inline frontend/src/App.jsx:56]: Why remove this check?

File: frontend/src/components/Page/page.cssmodule.scss [hunk 3]
```diff
@@ -7,6 +7,7 @@
 }
 
 .container {
+  position: relative;
   background-color: var(--w);
```
Comment (sandrica89) [inline frontend/src/components/Page/page.cssmodule.scss:11]: Has dark mode being handled?

File: frontend/src/components/Tutorial/Steps/billing.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
+import agent from 'services/agent';
+
+const steps = [
+  {
+    placement: 'center',
+    target: 'body',
+    title: 'Gérer mes facturations',
+    content: 'Cette page vous permet de gérér vos informations de facturation et de générer vos factures pour les Service de Santé Universitaire.',
```
Comment (sandrica89) [inline frontend/src/components/Tutorial/Steps/billing.js:8]:
```suggestion
    content: 'Cette page vous permet de gérér vos informations de facturation et de générer vos factures pour les Services de Santé Universitaire.',
```

File: frontend/src/components/Tutorial/Steps/billing.js [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    target: '#no-appointments',
+    shouldSkip: () => {
+      const now = new Date();
+      return agent.Appointment.get().then(appointments => appointments.some(appointment => {
+        const appointmentDate = new Date(appointment.appointmentDate);
+        return appointmentDate.getFullYear() === now.getFullYear()
+          && appointmentDate.getMonth() === now.getMonth();
+      }));
+    },
+    content: "Vous n'avez pas déclarer de séances pour ce mois-ci, il n'y a donc pas de facture à generer. Commencez par déclarer des séances depuis l'onglet dédié ou générez une facture pour un autre mois.",
```
Comment (sandrica89) [inline frontend/src/components/Tutorial/Steps/billing.js:27]:
```suggestion
    content: "Vous n'avez pas déclarer de séances pour ce mois-ci, il n'y a donc pas de facture à générer. Commencez par déclarer des séances depuis l'onglet dédié ou générez une facture pour un autre mois.",
```

File: migrations/20210928081959_tutorial.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table('psychologists', (table) => {
+    table.boolean('hasSeenTutorial');
```
Comment (sandrica89) [inline migrations/20210928081959_tutorial.ts:5]: We can set default to false explicitely and put it not null.
Won't we have any issue if the field remains null?

**Review by sandrica89 — CHANGES_REQUESTED** [review]
We could also add a cypress test just to ensure that we can start and skip the tuto? What do you think?

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**